scripts/reindex.py

```diff
@@ -70,6 +71,10 @@ def __init__(self, prog) -> None:
                     default=False,
                     action='store_true',
                     help='Purge the queues before taking any action on the index.')
+parser.add_argument('--nowait', '--no-wait',
+                    default=False,
+                    action='store_true',
+                    help='Don\'t wait for queues to empty before exiting script.')
```

Comment:
```suggestion
                    help="Don't wait for queues to empty before exiting script.")
```

This is a PEP-8 recommendation. I'm getting tired reminding you of this. So even though it should be implied, please add a separate commit in which you amend `CONTRIBUTING.rst` with a section about not using backslash to escape single (double) quotes in single (double) quoted strings. Link that section to specific PEP8 recommendation about that.

Reply:
Added contributing guideline regarding quote escaping after guideline about preference for single quoted strings.